### ISSUE TITLE
Custom Sprite Sheet Files

### ISSUE THREAD

**Issue by ikskuh** [MasterQ32/VersaTile#17]
Allow the creation of custom sprite sheet files that store
- the image
- the minimal tile size
- a preselection of tiles
- predefined sprite groups

**Comment by aggsol** [comment]
The TSX format of Tiled is a good example what properties might be useful
http://doc.mapeditor.org/reference/tmx-map-format/#tileset

**Comment by ikskuh** [comment]
Good idea! Spacing and Margin are really useful, the name as well ^^

Maybe this feature can be made into some kind of "in program spritesheet database" with predefined objects per sprite sheet.

**Comment by aggsol** [comment]
people might want to distribute it, maybe something simple like having to files with the same name `spritesheet.png` and `spritesheet.vss` wher `vss` stands for VersaTile Sprite Sheet ;-P

**Comment by ikskuh** [comment]
Yeah, sounds reasonable. Maybe it's possible to embed it into png metadata or something, this would be cool as well
